cra-network-aware-data-fetching/src/App.js

```diff
@@ -1,26 +1,14 @@
 import React from 'react';
-import logo from './logo.svg';
+
 import './App.css';
+import IFixitSearch from './containers/IFixitSearch';
 
-function App() {
+const App = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App-content'>
+      <IFixitSearch />
```

Comment:
Similar comment to earlier (iFixitSearch)

Reply:
The same reason as above.

Reply:
Gotcha. Similar to above, I'd say we can skip this change then.